### PR TITLE
Add pin definitions to target.json

### PR DESCRIPTION
**Pull request by bremoran**
These pin definitions will allow yotta-config to define pin names.

### CODE REVIEW

**Comment by 0xc0170** [comment]
What is the output from these pins? YOTTA_CONFIG_MBEDOS_PINS_xxx ? 
Are we using any external script?

Where are defined mcu k64f definitions like PTA1?

**Comment by bremoran** [comment]
This is a non-breaking change.

**Comment by bremoran** [comment]
@0xc0170 Please see the other related PRs ;)

File: target.json [hunk 1]
```diff
@@ -33,9 +72,6 @@
     "armv7-m",
     "arm",
     "gcc",
-    "lwip",
```
Comment (bogdanm) [inline target.json]: This looks like an unrelated change, are we sure it doesn't break anything?

Comment (bremoran) [inline target.json]: You're right, it's absolutely an unrelated change and it will break things.  I'll add another revision to fix it.

**Comment by autopulated** [comment]
+1 looks  good now.

**Comment by bogdanm** [comment]
Yup, +1
